src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -358,7 +358,9 @@ public void WriteBindings(IResource resource)
             // 2. Default endpoints added by the framework
             // 3. Explicitly added endpoints
             // But wherever they come from, we treat the first one as Default, for each scheme.
-            var schemesEncountered = new HashSet<string>();
+            var httpSchemesEncountered = new HashSet<string>();
+
+            static bool IsHttpScheme(string scheme) => scheme is "http" or "https";
```

Comment:
Ideally, we'd have a centralized helper for this check, as we make the same check for the env var injection (and possibly other places)